### ISSUE TITLE
Possible source for problems?

### ISSUE THREAD

**Issue by janpio** [philschatz/project-bot#24]
What happens if someone creates an issue and adds a label during creation - and I have a `new_issue` automation and a `added_label` automation for that label? (Similar source for problems with assignment, milestone etc.)

**Comment by philschatz** [comment]
Yes, this is unfortunately a bug. Currently project-bot does not notice other rules (like `has_label`) when an Issues is created.

I would like to fix it but will need to add automated tests since it would not be an isolated change (the unit tests are long overdue in this repo)

**Comment by janpio** [comment]
Ok, just wanted to confirm that there is no handling for that yet. 
I think this is not really a "bug", but more an unimplemented feature though - if you write that down it should be just fine for now.
